cmd/podman/common/create_opts.go

```diff
@@ -204,10 +204,10 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig, cgroup
 	for _, m := range cc.HostConfig.Mounts {
 		mount := fmt.Sprintf("type=%s", m.Type)
 		if len(m.Source) > 0 {
-			mount += fmt.Sprintf("source=%s", m.Source)
+			mount += fmt.Sprintf(",source=%s", m.Source)
 		}
 		if len(m.Target) > 0 {
-			mount += fmt.Sprintf("dest=%s", m.Target)
+			mount += fmt.Sprintf(",dst=%s", m.Target)
```

Comment:
Is the change from `dest` to `dst` deliberate, and if so does it merit a note in the commit message?

Reply:
it is delib, but for a one handed guy rn, i choose to not mention it